### PR TITLE
Added Q10 on the FAQ page (Issue2770)

### PR DESCRIPTION
**Pull request by anikx7**
Fixes # 2770

### Description
Added Q10 on the FAQ page
"Q10. How can I keep track of 10 step vetting progress?"

### Raw.Githack preview link
https://raw.githack.com/anikx7/anikx7.github.io/Issue2770/#!./pages/vi/vi-faq.md

### CODE REVIEW

**Comment by perryc85** [comment]
@anikx7 - Good job overall, but the numbers for each question are out of order. You have two questions that are labeled Q10.

**Comment by anikx7** [comment]
> 
> 
> @anikx7 - Good job overall, but the numbers for each question are out of order. You have two questions that are labeled Q10.

@perryc85 thank you for pointing this issue. I have fixes it.

File: pages/vi/vi-faq.md [hunk 1]
```diff
@@ -11,22 +11,22 @@
 - **Q7:** [If I'm an international student, how soon can I get my paperwork required by my visa?](#Q7:_If_I'm_an_international_student,_how_soon_can_I_get_my_paperwork_required_by_my_visa?)
 - **Q8:** [How many pull requests and issues in total do I need to make to finish the steps?](#Q8:_How_many_pull_requests_and_issues_in_total_do_I_need_to_make_to_finish_the_steps?)
 - **Q9:** [How long do I have to complete these steps?](#Q9:_How_long_do_I_have_to_complete_these_steps?)
-
+- **Q10:** [How can I keep track of 10 steps vetting progress?](#Q10:_How_can_I_keep_track_of_10_steps_vetting_progress?)
```
Comment (paulbert) [inline pages/vi/vi-faq.md]: Please add a blank line here.  It makes the markdown a little easier to follow when working on it.

**Comment by paulbert** [comment]
@anikx7 The blank line should be after **Q10** so the **Technical Questions** heading has a space above and below it.

**Pull request merged**